src/utils/string/githubPRLinkify.tsx

```diff
@@ -0,0 +1,11 @@
+import ReactDOMServer from "react-dom/server";
+import { StyledLink } from "components/styles/StyledLink";
+
+export const githubPRLinkify = (unlinkified: string) =>
+  unlinkified.replace(
+    /https:\/\/github.com\/[a-zA-Z0-9-]+\/[a-zA-Z0-9-]+\/pull\/\d+/gi,
+    (match) =>
+      ReactDOMServer.renderToString(
```

Comment:
This seems redundant. Why would we render this to a string and then convert it back to html here https://github.com/evergreen-ci/spruce/pull/878/files#diff-6ac1cd60dde9a2d56f1196499e04205d1e42f8ac7c96c7992b8b5763f814a700R138

Reply:
We should just return the jsx element

Reply:
I think this may be the best we can do with the `.replace` solution since the callback can only return a string. I utilized the renderToString in order to reuse the <StyledLink /> component. 
The alternate solution would be to step through the unlinkified string based on start/end indices for matched items and append JSX elements to an array to render. I prefer the current solution for simplicity.